sdk/eventhub/event-hubs/review/event-hubs.api.md

```diff
@@ -156,6 +156,9 @@ export interface LastEnqueuedEventProperties {
     sequenceNumber?: number;
 }
 
+// @public
+export const logger: import("@azure/logger").AzureLogger;
```

Comment:
I wonder why did we expose the logger.

Reply:
Discussed offline. The customer may want to override the default logger to enrich the logging experience with app-specific info.